frodo.js

```diff
@@ -2,89 +2,82 @@
  * Extras Plugin
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?